tests/provisioner-tests/Makefile

```diff
@@ -1,29 +1,17 @@
-IMG_NAME = provisioner-tests
-IMG = $(DOCKER_PUSH_REPOSITORY)$(DOCKER_PUSH_DIRECTORY)/$(IMG_NAME)
-TAG = $(DOCKER_TAG)
+APP_NAME = provisioner-tests
+APP_PATH = tests/provisioner-end
+BUILDPACK = eu.gcr.io/kyma-project/test-infra/buildpack-golang-toolbox:v20190913-65b55d1
+SCRIPTS_DIR = $(realpath $(shell pwd)/../..)/scripts
 
-.PHONY: ci-pr ci-master ci-release resolve build-and-test build-image push-image pull-licenses
+include $(SCRIPTS_DIR)/generic_make_go.mk
 
-ci-pr: resolve build-and-test build-image push-image
-ci-master: resolve build-and-test build-image push-image
-ci-release: resolve build-and-test build-image push-image
+verify:: vet
+
+$(eval $(call test-local))
```

Comment:
Do we need this line?
From what I understand `test-local: ;` is enough to disable the `test-local` rule.